packages/core/src/customizations/graphql/vtex/typeDefs/product.graphql

```diff
@@ -0,0 +1,6 @@
+extend type StoreProduct {
```

Comment:
Do we actually need this file here? Or is this just a sample? If this is just a sample I'd like this to be moved to the documentation.

Reply:
This is a sample. Involves the _scaffold since the clone vs. creating using documentation_ issue hahah.

Reply:
I removed the samples [here](https://github.com/vtex/faststore/pull/1903/commits/c3a3fa1eae487b847de7c2067d8372dae7905da8). Also documentation added [here](https://github.com/vtex/faststore/pull/1903/commits/574589df2b00b265413061501cc71ac1b7ba0b4a).